docs/release/support_matrix.md

```diff
@@ -1,7 +1,428 @@
+
+
+### Enterprise Linux 9 (EL9): RHEL 9, Rocky Linux 9, AlmaLinux 9
+
+DAOS Version 2.4.0 has not been validated and is not supported on EL9.
+It is a roadmap item to validate EL 9.2 (or later) with DAOS Version 2.6,
+and potentially with DAOS 2.4.1.
+
+Links to RHEL 9 Release Notes:
+
+* [RHEL 9.0](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/9.0_release_notes/index)
+* [RHEL 9.1](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/9.1_release_notes/index)
+* [RHEL 9.2](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/9.2_release_notes/index)
+
+Links to Rocky Linux Release Notes:
+
+* [Rocky Linux 9.0](https://docs.rockylinux.org/release_notes/9_0/)
+* [Rocky Linux 9.1](https://docs.rockylinux.org/release_notes/9_1/)
+* [Rocky Linux 9.2](https://docs.rockylinux.org/release_notes/9_2/)
+
+Links to AlmaLinux Release Notes:
+
+* [AlmaLinux 9.0](https://wiki.almalinux.org/release-notes/9.0.html)
+* [AlmaLinux 9.1](https://wiki.almalinux.org/release-notes/9.1.html)
+* [AlmaLinux 9.2](https://wiki.almalinux.org/release-notes/9.2.html)
+
+
+### Unsupported Linux Distributions
+
+With DAOS Version 2.4, CentOS 7 and RHEL 7 are no longer supported.
+Please update your DAOS servers to a supported EL8 level before updating to DAOS 2.4.
+
+DAOS also does not support
+openSUSE Tumbleweed,
+Fedora,
+CentOS Linux,
+CentOS Stream,
+Ubuntu, or
+Oracle Linux.
+
+
+## Operating Systems supported for DAOS Clients
+
+The DAOS software stack is built and supported on
+Linux for the x86\_64 architecture.
+
+In DAOS Version 2.4, the supported Linux distributions and versions for DAOS clients
+are identical to those for DAOS servers. Please refer to the previous section for details.
+
+In future DAOS releases, DAOS client support may be added for additional
+Linux distributions and/or versions.
+
+
+## Fabric Support
+
+DAOS Version 2.4 supports both OFI [libfabric](https://ofiwg.github.io/libfabric/).
+and UCF [UCX](https://openucx.org/) for communication in the DAOS data plane.
+This section describes the supported network providers and contains references
+to vendor-specific information for the supported networking hardware.
+
+
+### OFI libfabric
+
+With the exception of UCX for InfiniBand networks, OFI libfabric is the recommended
+networking stack for DAOS.  DAOS Version 2.4 requires at least version 1.18.0 of
+[libfabric](https://ofiwg.github.io/libfabric/).
+It is strongly recommended to use exactly the provided libfabric version
+on all DAOS servers and all DAOS clients.
+
+Links to libfabric releases on github
+(the RPM distribution of DAOS includes libfabric RPM packages with the correct version):
+
+* [libfabric 1.18.0](https://github.com/ofiwg/libfabric/releases/tag/v1.18.0) (Apr 08, 2023)
+* [libfabric 1.18.1rc1](https://github.com/ofiwg/libfabric/releases/tag/v1.18.1rc1) (release candidate)
```

Comment:
it might be best to only recommend 1.18.1rc1 given that 1.18.0 has some serious issues w/ tcp (thread-safety and busy spin).

Reply:
done